config.py

```diff
@@ -34,3 +34,14 @@ def set_config_file(cfg_file=None):
             raise IOError("No valid configuration files found.")
 
     return CONFIG
+
+
+"""Oslo_log setup"""
+from oslo_config import cfg
```

Comment:
Imports should go at the top of the file - I am surprised flake8 doesn't complain here?